diff --git a/PowerEditor/gcc/makefile b/PowerEditor/gcc/makefile
index 31c7104..b952009 100644
--- a/PowerEditor/gcc/makefile
+++ b/PowerEditor/gcc/makefile
@@ -76,13 +76,13 @@ ifeq "$(TARGET_CPU)" ""
 $(error TARGET_CPU detection failed)
 endif
 ifneq "$(filter-out x86_64 i686,$(TARGET_CPU))" ""
-$(error $(TARGET_CPU) build is unsupported)
+#$(error $(TARGET_CPU) build is unsupported)
 endif
 ifeq "$(TARGET_CPU)" "i686"
 # for some reason i686 versions of MinGW-w64 GCC don't include a linking library for SensApi.dll
 # thus it is generated on the fly, but first check if the DLL is available
 ifeq "$(wildcard $(WIN_DIR)/system32/SensApi.dll)" ""
-$(error $(TARGET_CPU) build requires "%windir%/system32/SensApi.dll" to be present)
+#$(error $(TARGET_CPU) build requires "%windir%/system32/SensApi.dll" to be present)
 endif
 # detect explicit definition of TARGET_CPU via command line to force a 32-bit build for MinGW-w64 with support multilib
 ifeq "$(origin TARGET_CPU)" "command line"
